app/components/AsciidocBlocks/Listing.tsx

```diff
@@ -40,6 +40,26 @@ hljs.registerLanguage('tla', function (hljs) {
   }
 })
 
+hljs.registerLanguage('oxql', function (hljs) {
+  return {
+    keywords: {
+      keyword: 'get join align filter group_by',
+    },
+    contains: [
+      hljs.QUOTE_STRING_MODE,
+      hljs.COMMENT('\\(\\*', '\\*\\)'),
+      hljs.COMMENT('\\\\\\*', '$'),
+      { begin: /\/\\/ }, // relevance booster
```

Comment:
I copied these 4 lines from the other one, but a) I'm not sure what the relevance booster line even is, and b) I don't know if oxql has comments.

Reply:
Nope, no concept of comments in OxQL